docs/standard/serialization/binaryformatter-migration-guide/winforms-migration-guide.md

```diff
@@ -0,0 +1,89 @@
+- `decimal`
+- `double`
+- `int`
+- `sbyte`
+- `float`
+- `TimeSpan`
+- `DateTime`
+- `uint`
+- `string`
+- `nint`
+- `nuint`
+- `long`
+- `ulong`
+
+Windows Forms also supports the following additional types:
+
+- `PointF`
+- `RectangleF`
+- `Bitmap`
+- `ImageListStreamer`
+
+### Clipboard
+
+All Standard OLE [`DataFormats`](/dotnet/api/system.windows.forms.dataformats#fields) don't go through `BinaryFormatter`, except for `DataFormats.Serializable` and any custom format. If you're using `DataFormats.Serializable` or a custom format, `BinaryFormatter` is used if your clipboard scenario involves types that aren't intrinsically handled when [`Clipboard.SetData`](/dotnet/api/system.windows.forms.clipboard.setdata) is called with your type and when [`Clipboard.GetData`](/dotnet/api/system.windows.forms.clipboard.getdata) is called to get your type. `BinaryFormatter` is also used if [`Clipboard.SetDataObject(object, copy: true)`](/dotnet/api/system.windows.forms.clipboard.setdataobject) is called. With the `BinaryFormatter` removal, you won't see an exception when setting the data on the clipboard if `BinaryFormatter` was needed. Instead, you'll see a string about `BinaryFormatter` being removed when you attempt to get the type that isn't intrinsically handled from the clipboard.
+
+### Drag-and-drop feature
+
+If your drag-and-drop scenario involves types that aren't intrinsically handled during serialization and deserialization, `BinaryFormatter` is used when [`Control.DoDragDrop`](/dotnet/api/system.windows.forms.control.dodragdrop) is called, and when [`DataObject.GetData`](/dotnet/api/system.windows.dataobject.getdata) is called to retrieve a type that's been dragged out of process. With the `BinaryFormatter` removal, you'll now see a string about `BinaryFormatter` being removed when you drop a dragged item in another process for drag-and-drop operations with types that aren't intrinsically handled.
+
+### The Windows Forms Designer
+
+The Windows Forms Designer also uses `BinaryFormatter` internally for ResX serialization and deserialization, and for CodeDom.
+
+Types and properties might participate in serialization without you realizing due to the standard behavior of the Windows Forms Designer. One way that `BinaryFormatter` is used that you might not be aware of is when a property on a <xref:System.Windows.Forms.UserControl> is introduced and that control is already populated at design time. It's likely that that data gets serialized into resource files. Consider the following conditions:
+
+- A property contains data at the time when a Form in the Designer is saved.
+- That property is not attributed with `DesignerSerializationVisibility(false)`.
+- That property is not read-only.
+- That property does not have a DefaultValueAttribute.
+- That property does not have a respective `bool ShouldSerialize[PropertyName]` method that returns `false` at the time of the CodeDOM serialization process. (Note: the method can have `private` scope.)
```

Comment:
Property type does not have a DesignTime serializer - https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.modeling.dsldefinition.designerserializer?view=visualstudiosdk-2019

Reply:
I added another bullet, does this sound about right?

Reply:
> I added another bullet, does it sound about right?

I don't see this change, did it get lost in merges?